src/components/ui/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const Avatar = ({classes}) => {
+    return (
+        <div className={classes}>
+            <svg xmlns="http://www.w3.org/2000/svg" width="36" height="37" viewBox="0 0 36 37" fill="none">
```

Comment:
Is this avatar intended just for testing, or are you planning to use it in the final implementation?"

Reply:
Ok, thanks for the feedback!